packages/components/src/components/password-input/password-input.tsx

```diff
@@ -1,15 +1,20 @@
 import classNames from 'classnames';
 import React from 'react';
-import PropTypes from 'prop-types';
 import Icon from '../icon';
-import Input from '../input';
+import Input, { TInputProps } from '../input';
+
+type TPasswordInput = Partial<TInputProps> & {
```

Comment:
Used `Partial` to make types for `Input` optional since there are some components that uses `PasswordInput` without passing the required props for `Input`

Reply:
I can see that the `type` is required in the InputField. If you put Partial it might accept even the `type` is undefined. If you remove Partial can i know what error will be return to you?

Reply:
If `TInputProps` is not wrapped with `Partial`, there will be some places that uses `PasswordInput` who does not pass in the required `type` prop, thus causing the build to fail. Since I wasn't so sure if the type definitions for `Input` component `type` needs to be made optional in `input.tsx`, I just made it optional for `PasswordInput` as for now
<img width="942" alt="Screenshot 2023-01-10 at 11 56 48 AM" src="https://user-images.githubusercontent.com/103016120/211458683-fd9c1096-46e4-4d44-888f-7cdad99f6adc.png">